app/services/dashboard/data.rb

```diff
@@ -1,17 +1,21 @@
 class Dashboard::Data
-  attr_reader :filter
+  attr_reader :filter, :settings
+
   # returns a status hash
   def self.status(filter = "")
     new(filter).report
   end
 
-  def initialize(filter = "")
+  def initialize(filter = "", settings: {})
```

Comment:
I guess this should have been `settings = {}`, or did you want to combine keywords with positional args? I'm surprised ruby was OK with this